addons/modules/functions/fnc_moduleCrewToGunner.sqf

```diff
@@ -0,0 +1,48 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function to have AI crew take the place of an incapacitated gunner.
+ *
+ * Arguments:
+ * 0: Logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC] call zen_modules_fnc_moduleCrewToGunner
+ *
+ * Public: No
+ */
+
+params ["_logic"];
+
+private _vehicle = attachedTo _logic;
+deleteVehicle _logic;
+
+if (isNull _vehicle) exitWith {
+    [LSTRING(NoObjectSelected)] call EFUNC(common,showMessage);
+};
+
+if !(alive _vehicle) exitWith {
+    [LSTRING(OnlyAlive)] call EFUNC(common,showMessage);
+};
+
+if (fullCrew [_vehicle, "gunner", true] isEqualTo []) exitWith {
+    [LSTRING(MustHaveGunner)] call EFUNC(common,showMessage);
+};
+
+_vehicle allowCrewInImmobile true;
+private _gunner = gunner _vehicle;
+private _newGunners = crew _vehicle;
```

Comment:
This can return dead units.

Reply:
https://github.com/zen-mod/ZEN/pull/423/commits/979a223c0ad159222d8ed3c6181ce5f073e54b13